src/Identifier/Normalizer/IdentifierDenormalizerInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Identifier\Normalizer;
+
+use ApiPlatform\Core\Exception\InvalidIdentifierException;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+interface IdentifierDenormalizerInterface extends DenormalizerInterface
```

Comment:
Do we really need this new interface? I would use the Symfony one directly.

Reply:
Oh okay I remember. This was only to document that the `denormalize` method throws an `InvalidIdentifierException` haha. I can remove it though,

Reply:
Yes please :)